### DISCUSSION TITLE
招收安保人员

### DISCUSSION THREAD

**Discussion by Doctor-Bright** [share121/inter-knot#3349] [其他]
### 招收设施内安保人员

***

# 首先：**感谢你的来访**

***

## 你的任务

>看守特定的区域（会有人与你交替岗位）

***

      在开始之前你需要了解一下基本信息

***
# SCP档案（图片文件）
![label895](https://github.com/user-attachments/assets/1092c625-f7fb-49c9-bc3b-a67eac58e8e3)
![doc_895](https://github.com/user-attachments/assets/210e6285-764e-4c89-a275-516564090b41)

***

     由于该异常的特性，需要有较多人手参与监视。
     你只需要在收容区入口的区域巡视。——因为向下前进通常是不被允许的。

***
## 注意事项

>1. 在工作中如果出现任何异常（如：突如其来的抑郁或自残自杀倾向。） 请立即汇报并原地等待。 
>2. 你不需要特别专注，只保证不会有可疑人员闯入就行。 
>3. 如果你不想继续的话，随时可以提出辞职申请。 

***
## 结算

>1. 在提交辞呈后会先对您进行精神评估，在确认没有异常后薪水会打到您的账户上（可以要求领取现金）。 

>2. 离开前会对您进行<u>E级记忆删除</u>，此举是为了防止敏感信息泄露，不会对您之后的生活造成影响。 

***

         此消息已开启访问监控

# 最后——祝您生活愉快。
### ——基金会人事部
***
![scplogo](https://github.com/user-attachments/assets/4f890803-59c0-48ec-b3a1-1b0af32e9ea0)

**Comment by StrinovaMaster** [comment]
你好，我可以来应聘吗？

**Reply by Doctor-Bright** [reply]
welcome

**Comment by yui-hanasaka** [comment]
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >会去被安排去看173吗（

**Reply by Doctor-Bright** [reply]
我们不反对您主动申请（）

**Comment by liubaiqing** [comment]
内容没看，但是我是传奇绳斤，这个委托我无脑接了
